docs/kubernetes/getstarted.md

```diff
@@ -0,0 +1,604 @@
+```
+
+Let's connect to the administrator interface through the wordpress NodePort service that we have just created. Let's first retrieve the port open for that service on the nodes
+
+```
+$ kubectl get -o jsonpath="{.spec.ports[0].nodePort}" services wordpress
+31004
+```
+
+We can browse any nodes url on port 31004 to find the wordpress website e.g. [http://173.30.3.2:31004/](http://173.30.3.2:31004/)
+and after some setup screens you will access your articles
+
+![wordpress first article](ressources/wordpress/wordpress.png)
+
+### Helm charting
+
+K3s comes with helm support built-in. Let's try to deploy prometheus and grafana monitoring of the cluster through HELM charts.
+
+```
+$ kubectl create namespace mon
+$ helm install --namespace mon --skip-crds  my-release stable/prometheus-operator
+$ helm list
+NAME            NAMESPACE       REVISION        UPDATED                                 STATUS          CHART                      APP VERSION
+my-release      mon             1               2020-02-04 16:08:41.70990744 +0100 CET  deployed        prometheus-operator-8.5.11 0.34.0
+$ kubectl config set-context --current --namespace=mon
+$ kubectl get po
+NAME                                                     READY   STATUS    RESTARTS   AGE
+my-release-kube-state-metrics-778b4d9786-tqp9r           1/1     Running   0          7m34s
+my-release-prometheus-node-exporter-xfdgv                1/1     Running   0          7m35s
+my-release-prometheus-node-exporter-ngzb4                1/1     Running   0          7m35s
+my-release-prometheus-node-exporter-lvmp8                1/1     Running   0          7m35s
+my-release-prometheus-oper-operator-69cc584dfb-lxjwp     2/2     Running   0          7m34s
+alertmanager-my-release-prometheus-oper-alertmanager-0   2/2     Running   0          7m21s
+my-release-grafana-6c447fc4c8-zkc4x                      2/2     Running   0          7m34s
+prometheus-my-release-prometheus-oper-prometheus-0       3/3     Running   1          7m10s
+```
+
+Let's connect to the grafana interface through the deployment that we have just created.
+
+We setup port forwarding to listen on port 8888 locally, forwarding to port 3000 in the pod selected by the deployment my-release-grafana
+
+```
+$ kubectl port-forward deployment/my-release-grafana  8888:3000
+Forwarding from 127.0.0.1:8888 -> 3000
+Forwarding from [::1]:8888 -> 3000
+```
+
+We can browse localhost on port 8888 to find the grafana UI e.g. [http://localhost:8888/](http://localhost:8888/)
+The username is `admin` and the default admin password to log into the grafana UI is `prom-operator`
+Then you can for instance import a dashboard and use the ID 8588 and don't forget to select the prometheus data source
+![kubernetes-deployment-statefulset-daemonset-metrics dashboard](ressources/grafana/grafana1.png)
+or the dashboard ID 6879
+![analysis-by-pod](ressources/grafana/grafana2.png)
+
+## Storage
```

Comment:
I think everything after this line should be moved out of the getting started. This is very specific and more deep topics that should not be part of a getting started.

Reply:
created an advanced doc and added a link to it